packages/testing/package.json

```diff
@@ -43,10 +43,8 @@
   ],
   "dependencies": {
     "@esm-bundle/chai": "^4.3.4-fix.0",
-    "@open-wc/chai-dom-equals": "^0.12.36",
     "@open-wc/semantic-dom-diff": "^0.20.0",
     "@open-wc/testing-helpers": "^2.3.1",
-    "@types/chai": "^4.2.11",
```

Comment:
Is this really unused?

Reply:
We only use the `@types/chai-dom` types, so yes :)